cmd/neofs-node/netmap.go

```diff
@@ -361,6 +360,33 @@ func (c *cfg) SetNetmapStatus(st control.NetmapStatus) error {
 	return c.updateNetMapState(func(*nmClient.UpdatePeerPrm) {})
 }
 
+func (c *cfg) ForceMaintenance() error {
+	return c.setMaintenanceStatus(true)
+}
+
+func (c *cfg) setMaintenanceStatus(force bool) error {
+	netSettings, err := c.cfgNetmap.wrapper.ReadNetworkConfiguration()
```

Comment:
Why have you decided to check network settings here and not in the control service?

Reply:
It is a Control service, I just distinguish handling of the request payload from message processing (signatures and so on).

Reply:
> It is a Control service

what do you mean?

Reply:
The method implements service needs.